src/reagent/cookies.cljs

```diff
@@ -11,15 +23,29 @@
    :domain - domain of the cookie, when null the browser will use the full request host name
    :secure? - boolean specifying whether the cookie should only be sent over a secure channel
    :raw? - boolean specifying whether content should be stored raw, or as EDN
+   :same-site - A keyword of either :strict, :lax, or :none (defaults to :none). Only supported when `supports-same-site?` is true
   "
-  [k content & [{:keys [max-age path domain secure? raw?]} :as opts]]
+  [k content & [{:keys [max-age path domain secure? raw? same-site] :as opts}]]
   (let [k (name k)
         content (if raw?
                   (str content)
                   (pr-str content))]
-    (if-not opts
+    (cond
+      (not (dissoc opts :raw?))
       (.set goog.net.cookies k content)
-      (.set goog.net.cookies k content (or max-age -1) path domain (boolean secure?)))))
+
+      @legacy-setter?
+      (.set goog.net.cookies k content (or max-age -1) path domain (boolean secure?))
+
+      :else
+      (.set goog.net.cookies k content #js{:maxAge   (or max-age -1)
+                                           :path     path
+                                           :domain   domain
+                                           :sameSite (condp = same-site
+                                                       :strict (.-STRICT goog.net.Cookies.SameSite)
+                                                       :lax (.-LAX goog.net.Cookies.SameSite)
+                                                       (.-NONE goog.net.Cookies.SameSite))
```

Comment:
i was now searching an hour for the problem, why my cookies won't be set in the browser. on firefox you will get an error in the console, but chrome does not tell you a thing.  
so the 'none' option does not work anymore in chrome.  
returning an error or just using :lax as a default, might be a better option.